include/SFML/Network/Ftp.hpp

```diff
@@ -484,7 +484,7 @@ class SFML_NETWORK_API Ftp : NonCopyable
     /// \see download
     ///
     ////////////////////////////////////////////////////////////
-    Response upload(const std::string& localFile, const std::string& remotePath, TransferMode mode = Binary);
+    Response upload(const std::string& localFile, const std::string& remotePath, TransferMode mode = Binary, const append = false);
```

Comment:
We never use `const` for function parameters passed by copy. And `bool` is missing from the declaration (did you really compile this code??).